### ISSUE TITLE
incendo/cloud time parameter

### ISSUE THREAD

**Issue by ParadauxIO** [oscailte-mc/discipline#16]
Part of our functionality (-e) is waiting on cloud to merge a PR by @Aberdeener available at https://github.com/Incendo/cloud/pull/330
This issue is to track changes to that PR and watch for its (hopefully) eventual merge so we can begin to use it. Chances are the functionality will not need to be implemented separetely by us as we won't get to that particular feature until it is done. 

In the event that it isn't, we will simply do the parsing outside of cloud until such a time that it is available.

**Comment by Aberdeener** [comment]
Yep :)
Adding the parser class to our codebase is a fine solution in the event it's not merged into cloud

**Comment by ParadauxIO** [comment]
The linked PR has been merged, planned for release in Cloud 1.7.0.
Apologies to the team at Incendo if we bothered them by arguing about the parser's semantics 😅

**Comment by Aberdeener** [comment]
1.7.0 was released, this can be closed
